crates/ef-testing/src/evm_sequencer/mod.rs

```diff
@@ -76,11 +79,25 @@ impl KakarotSequencer {
                     .map_err(|err| StateError::ProgramError(ProgramError::Parse(err)))?,
             )?),
         )?;
+        (&mut self.0.state).set_contract_class(
+            &FEE_TOKEN_CLASS_HASH,
+            ContractClass::V0(ContractClassV0::try_from_json_string(
```

Comment:
why do we need this?

can't we store FEE_TOKEN_CLASS as a ContractClass already?

Reply:
so I'm using this serde conversion because conversion between starknet-rs types and blockifier types is complicated. I need to use the starknet-rs type in order to have access to the `class_hash()` function.

Reply:
I see, very cursed